specification/definitions/ServiceContract.json

```diff
@@ -0,0 +1,190 @@
+{
+  "type": "object",
+  "required": [
+    "type"
+  ],
+  "properties": {
+    "type": {
+      "$ref": "../types/ServiceContract.json"
+    },
+    "id": {
+      "$ref": "../includes/read-only-uuid.json"
+    },
+    "attributes": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+      }
```

Comment:
Empty properties?

Reply:
`attributes` can indeed be removed, because it has no properties.